voicevox_engine/tts_pipeline/tts_engine.py

```diff
@@ -374,6 +426,143 @@ def synthesize_wave(
         wave = raw_wave_to_output_wave(query, raw_wave, sr_raw_wave)
         return wave
 
+    def get_sing_phoneme_and_f0_and_volume(
```

Comment:
これがTTSEngineにあるのは不思議なので、とりあえずFIXME付けとくのはどうでしょう。
```suggestion
    # FIXME: sing用のエンジンに移すかクラス名変える
    def get_sing_phoneme_and_f0_and_volume(
```

Reply:
私も実装しながら`TTSEngine`にこれがあるのはおかしいなと思ったのですが、エンジンクラスを分けた場合にはコアを複数のクラスに渡してあげる必要があり、同じコアが複数のエンジンに内包されているのは設計的におかしさを感じたので、一旦TTSEngineに実装しました。
`SynthesisEngine`に戻すというのも一つアリかもしれません。

とりあえずFIXMEで対処しようと思います。